### ISSUE TITLE
Usage of fine network

### ISSUE THREAD

**Issue by byungjur96** [Ruyi-Zha/naf_cbct#21]
Hi, I was running some experiments with NAF framework, and I was wondering the usage of fine network.
From my knowledge, fine network is a concept that was introduced in NeRF.
In config file in NAF, there exists some options about fine network such as n_fine.
However, I could not find exact usage of fine networks. For example, in loss function.
Is there anything I'm missing about fine networks
Thanks in advance!

**Comment by Ruyi-Zha** [comment]
Hi thanks for your interest in our work.

We actually don't use the fine network, just one single network. Since our code is adapted from NeRF, we keep relevant code and hyperparameters such as `n_fine` for future development.

Hope this helps.

**Comment by byungjur96** [comment]
Thanks for your quick response!
